.az/mshost.yaml

```diff
@@ -317,6 +317,7 @@ jobs:
 
       - script: |
           call activate pyenv
+          $env:vcvars="$env:VCINSTALLDIR\Auxiliary\Build\vcvars64.bat"
```

Comment:
Might be good to add a comment saying we do this so cmake comes from VS

Reply:
yes. I will put some comments here after the tryout of getting where VS is installed